lib/govspeak.rb

```diff
@@ -296,7 +296,7 @@ def render_image(image)
     extension("example", surrounded_by("$E")) do |body|
       <<~BODY
         <div class="example" markdown="1">
-          #{body.strip.gsub(/\A^\|/, "\n|").gsub(/\|$\Z/, "|\n")}
+        #{body.strip.gsub(/\A^\|/, "\n|").gsub(/\|$\Z/, "|\n")}
```

Comment:
This is a super good find. Would you mind just fixing the commit message over run issue you have and add a note in there about why this works please.

Reply:
Commit message fixed, and some explanation about the issue that Bruce found added above